omod/src/main/resources/config.xml

```diff
@@ -111,6 +111,29 @@
 		<defaultValue>false</defaultValue>
 		<description>Whether email should be sent for tele-consultation</description>
 	</globalProperty>
+	<globalProperty>
+		<property>bahmni.appointment.adhocTeleConsultation.patientEmailNotificationSubject</property>
+		<defaultValue>AdhocTeleconsultation started</defaultValue>
+		<description>Subject that will be used in the mail sent to patients for AdhocTeleconsultation</description>
+	</globalProperty>
+	<globalProperty>
+		<property>bahmni.appointment.adhocTeleConsultation.patientEmailNotificationTemplate</property>
+		<defaultValue>
+			Hi {0}, {1} has started an adhoc tele-consultation with you! Please use this link {3}. If you have any questions, please reach out to administration for assistance. See you soon!

```

Comment:
Same comment. No need for the word "adhoc".

Reply:
Updated